packager.go

```diff
@@ -441,14 +447,18 @@ func (this *Packager) ServeHTTP(resp http.ResponseWriter, req *http.Request, par
 		return
 	}
 	now := time.Now()
+	validityHRef, err := url.Parse("/" + ValidityMapURL)
```

Comment:
nit: if we're parsing a url by prefixing with a `/` then is it really a url? maybe it should be `validityURL, err := url.Parse("/" + ValidityMapPath)` or something.

Reply:
Met you halfway; renamed ValidityMapURL to Path, but left validityHRef alone, to indicate it's a relative URL.